geth/node/node_api_test.go

```diff
@@ -0,0 +1,54 @@
+package node
+
+import (
+	"testing"
+
+	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
+	"github.com/status-im/status-go/geth/params"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWhisperLightModeEnabledSetsEmptyBloomFilter(t *testing.T) {
+	config, err := params.NewNodeConfig("/tmp", params.StatusChainNetworkID, true)
+	require.NoError(t, err)
+	config.WhisperConfig.LightClient = true
+
+	// Node is ephemeral (only in memory).
+	node, nodeErr := MakeNode(config)
+	require.NoError(t, nodeErr)
+	require.NoError(t, node.Start())
+	defer func() {
+		err := node.Stop()
+		require.NoError(t, err)
+	}()
+
+	var whisper *whisper.Whisper
+	err = node.Service(&whisper)
+	require.NoError(t, err)
+
+	bloomFilter := whisper.BloomFilter()
+	expectedEmptyBloomFilter := make([]byte, 64)
+	require.NotNil(t, bloomFilter)
+	require.Len(t, bloomFilter, len(expectedEmptyBloomFilter))
+	require.Equal(t, bloomFilter, expectedEmptyBloomFilter)
```

Comment:
I believe it is the other way around. `expected` goes first.

Reply:
I think I got confused by the `Len` method signature and just applied the same order :-)